config/default_records/request_types/004_limber_high_throughput.yml

```diff
@@ -199,3 +199,10 @@ limber_targeted_nanoseq:
     - LTN Cherrypick
   library_types:
     - Targeted Nanoseq
+limber_rnaf:
+  <<: *limber_htp_library
+  name: Limber RNAF
+  acceptable_purposes:
+    - LBR Cherrypick
+  library_types:
+    - RNA FFPE
```

Comment:
"May need to inform NPG about this" - reminder - has this been done?

Reply:
Nope, good reminder thanks.